webapp/packages/core-blocks/src/Tree/TreeNode/TreeNodeExpand.tsx

```diff
@@ -86,13 +87,16 @@ export const TreeNodeExpand = observer<Props>(function TreeNodeExpand({ leaf, bi
   }
 
   return (
-    <div
+    <Clickable
+      as="div"
       className={s(styles, { treeNodeExpand: true, expanded: context.expanded, big }, className)}
+      focusable={!disabled && expandable}
+      disabled={disabled || !expandable}
```

Comment:
looks a bit strange, is Clickable is focusable by default? if so then disabled items usually can't be focused (so disabled prop should be enough)